.github/scripts/ci-style.sh

```diff
@@ -1,6 +1,6 @@
 . $(dirname "$0")/ci-common.sh
 
-export RUSTFLAGS="-D warnings"
+export RUSTFLAGS="-D warnings -A unknown-lints"
```

Comment:
Added this because in GitHub CI clippy version is older than on my PC, this is needed for `#[allow(clippy::needless_pass_by_ref_mut)]`